service/src/test/java/com/commercetools/pspadapter/payone/notification/common/PaidNotificationProcessorTest.java

```diff
@@ -145,7 +155,7 @@ public void whenChargeTransactionIsSuccess_ChangeTransactionStateNOTAdded() thro
         Payment paymentAuthSuccess = testHelper.dummyPaymentOneChargeSuccess20Euro();
         Transaction firstTransaction = paymentAuthSuccess.getTransactions().get(0);
         notification.setSequencenumber(firstTransaction.getInteractionId());
-        ImmutableList<UpdateAction<Payment>> authSuccessUpdates = testee.createPaymentUpdates(paymentAuthSuccess, notification);
+        List<UpdateAction<Payment>> authSuccessUpdates = testee.createPaymentUpdates(paymentAuthSuccess, notification);
```

Comment:
There is a replacement in case it makes sense: `Collections.unmodifiableList(list)`

Reply:
Not possible as the list is modified in subclasses... - actually there's no need to use unmodifiable lists.